### PR TITLE
ODH Kueue: Setup OpenShift CI

### PR DESCRIPTION
**Pull request by ChughShilpa**
Fix [RHOAIENG-1438](https://issues.redhat.com/browse/RHOAIENG-1438)

### CODE REVIEW

**Comment by ChughShilpa** [comment]
e2e tests are passing now. Could you once review it ? @sutaakar

File: ci-operator/config/opendatahub-io/kueue/opendatahub-io-kueue-dev.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    cluster_profile: aws-opendatahub
+    env:
+      BASE_DOMAIN: openshift-ci-aws.rhaiseng.com
+      HYPERSHIFT_AWS_REGION: us-west-2
+      HYPERSHIFT_NODE_COUNT: "3"
+    test:
+    - as: e2e-odh-kueue
+      cli: latest
+      commands: |
+        oc apply --server-side -k "github.com/ChughShilpa/kueue/config/rhoai?ref=dev"
```
Comment (sutaakar) [inline ci-operator/config/opendatahub-io/kueue/opendatahub-io-kueue-dev.yaml:31]: Can you install Kueue using ODH, referring to the ref from PR check or main branch?

Something similar as done in https://github.com/openshift/release/blob/master/ci-operator/config/opendatahub-io/codeflare-operator/opendatahub-io-codeflare-operator-main.yaml#L45-L53

That way you can test deployment scripts from PR checks (in future also image built from the source code, once image placeholder is supported in ODH).

Comment (ChughShilpa) [reply ci-operator/config/opendatahub-io/kueue/opendatahub-io-kueue-dev.yaml]: Added the change @sutaakar

**Pull request merged**